### ISSUE TITLE
Define case-sensitivity for models

### ISSUE THREAD

**Issue by kevinburkeshyp** [balderdashy/sails#2804]
As far as I can tell, your options for enabling case-sensitivity are 

a) define it on the adapter, which is shared by all of your database tables
b) define it on a per-query basis

Given we have already created a ton of indexes to hit the lowercased query, changing the adapter to be case sensitive is a nonstarter. Per-query is also annoying because you need to define it for each query a table uses.

Any way to set a `caseSensitive` attribute on a model object? that way queries using that table would be case sensitive; it seems like the right unit of on/off-ness for case sensitivity.

**Comment by kevinburkeshyp** [comment]
actually now that I double check it looks like the syntax proposed here: https://github.com/balderdashy/waterline/issues/239 does not work, the generated query is

```sql
SELECT ... WHERE LOWER("table"."driverId") undefined AND "table"."driverId" undefined
```

**Comment by sailsbot** [comment]
Thanks for posting, @kevinburkeshyp.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
